src/main/java/org/altbeacon/beacon/notifier/RegionMonitorNotifier.java

```diff
@@ -0,0 +1,66 @@
+package org.altbeacon.beacon.notifier;
+
+import org.altbeacon.beacon.MonitorNotifier;
+import org.altbeacon.beacon.Region;
+
+import java.util.ArrayList;
+
+/**
+ * Created by maxence on 17/04/15.
+ */
+public abstract class RegionMonitorNotifier extends ArrayList<Region> implements MonitorNotifier
```

Comment:
Interesting implementation.   This would allow the RegionMonitorNotifier to exist alongside the existing MonitorNotifier.